### DISCUSSION TITLE
Toast to the OSCR Launch - 8/12/24

### DISCUSSION THREAD

**Discussion by BekahHW** [open-sauced/feedback#40] [Show and tell]
We're continually trying to find new ways to bring value to our platform, and last week we launched the OSCR (Open Source Contributor Rating) to help bring greater visibility to the work of open source contributors. Bringing the OSCR to life has been a journey in itself, and I'm really excited to see how far we've brought it today and to see the stories that are coming as a result of it. 

Read about the OSCRs:
- [docs](https://opensauced.pizza/docs/features/contributor-insights/#open-source-contributor-rating-oscr)
- [Introducing the OSCR: Celebrating Open Source Stars](https://opensauced.pizza/blog/introducing-OSCR)

## How to toast

This is the fun part. Take a selfie with a pizza or drink, and post as a comment below with a shout out to someone or the team that made this happen.

Here's to the OSCR, and to all the incredible open source contributors out there! May it help amplify your efforts and inspire even more collaboration and support in open source. Cheers!

**Comment by BekahHW** [comment]
I honestly can't not name everyone on the team. Everyone had a hand in making this a success, from beautiful new designs, to developing the story of the OSCR, to seeing it implemented on the Repo pages, Contrib Insights pages, DevCards, and profiles.

Cheers to @brandonroberts, @nickytonline, @bdougie, @zeucapua, @isabensusan, @chhristopher, and @jpmcb for amazing work 🎉 

![IMG_7960](https://github.com/user-attachments/assets/e1d897b3-c486-46d0-b156-a03259eb12d7)

**Comment by bdougie** [comment]
Awesome job @BekahHW leading the effort to get this shipped and continued to get this shipped. 

- @zeucapua for leading the majority of the visual updates along with a continued stream of updates https://github.com/open-sauced/app/pull/3879 https://github.com/open-sauced/app/pull/3857
- @nick great job with the OG images https://github.com/open-sauced/app/pull/3903
- @brandon for shipping the oscr.me redirects today! https://github.com/open-sauced/app/pull/3973
- @isabensusan for the design updates to the cards https://github.com/open-sauced/app/issues/3872
- @john for the API updates and getting Sway as pillar of this rating https://github.com/open-sauced/app/pull/3829

![Screenshot 2024-08-20 at 10 12 20 AM](https://github.com/user-attachments/assets/81b554d8-e339-4339-add0-46ff3035a6d0)

_Drinking ginger and tumeric tea._

**Comment by jpmcb** [comment]
From both me and Nicolas Cage, great work everyone!!!!!! 🎉 🎉 

It's amazing to see how fast we've been able to turn around these features and get them shipped for our users. Great work to @BekahHW, as always, on captaining us through this launch and producing some stellar content. Hats off to the whole @open-sauced/engineering team on cutting _alot_ of code to make this all work - even though I was OOO towards the end of last week, it was awesome checking in and seeing the momentum continue! Cheers!

![Screenshot 2024-08-20 at 5 03 57 PM](https://github.com/user-attachments/assets/4daccb8d-ba46-4c37-9ee9-a94515cb7a01)

_(ps, my mug says "You're my national treasure")_

**Comment by zeucapua** [comment]
I'm honored to be part of such an amazing team, launching an amazing feature! Great work everyone!!!

From the designs to the code to the content, OSCR has made an impact bigger than what I imagined. Shoutout to @BekahHW for captaining the launch to success, @isabensusan for the great DevCard designs, the @open-sauced/engineering for getting the feature out so quickly in so many parts of the app, and everyone sharing the feature off through stories and social media posts everywhere. 

Let's keep this momentum up and ship some more! 🚀🍕✨

![Photo on 8-21-24 at 12 16](https://github.com/user-attachments/assets/c74c3dc0-26ec-4906-a823-0be81790bf5c)

_from totally real Apple HQ_

**Reply by jpmcb** [reply]
Say hi to Tim for us!!! 🍎

**Comment by brandonroberts** [comment]
Shoutout to the team and all the people who generated the new DevCards during the OSCR launch! It was great seeing everything come together during the launch and we learn something new with each one! 🍕 

![Photo on 8-21-24 at 10 28 AM #3](https://github.com/user-attachments/assets/0edcdcb2-a966-4d5c-956d-0f4ecb7015fb)

**Comment by nickytonline** [comment]
Big ups to the whole crew: @brandonroberts, @BekahHW, @bdougie, @zeucapua, @isabensusan, @chhristopher, and @jpmcb.

And big ups to @BekahHW for being the captain for this launch!

![CleanShot 2024-08-26 at 18 29 14](https://github.com/user-attachments/assets/358ab978-d074-4b81-9dac-5671ee6c08c1)
